plugins/lua/building-hacks.lua

```diff
@@ -15,19 +15,30 @@ local _ENV = mkmodule('plugins.building-hacks')
             action -- a table of number (how much ticks to skip) and a function which gets called on shop update
             canBeRoomSubset -- room is considered in to be part of the building defined by chairs etc...
```

Comment:
rooms no longer exist. it's all zones now. the implementation may not need to change, but the docs should be updated